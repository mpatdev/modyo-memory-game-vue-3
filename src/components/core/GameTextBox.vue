<script setup>
import useInput from '@/composables/useInput'

const { model } = useInput()
</script>

<template>
  <input
    v-model="model"
    class="game-text-box"
  >
</template>

<style>
.game-text-box {
    @apply border-b border-gray-400 outline-none focus:border-blue-300 w-full;
}
</style>