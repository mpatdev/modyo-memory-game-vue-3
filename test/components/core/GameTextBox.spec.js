import GameTextBox from '@/components/core/GameTextBox.vue'

import snapshotTest from '@test/snapshotTest'

snapshotTest(GameTextBox, 'GameTextBox')