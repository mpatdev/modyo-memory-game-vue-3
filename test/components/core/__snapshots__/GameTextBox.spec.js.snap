// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GameTextBox > Check Snapshot 1`] = `"<input class=\\"game-text-box\\">"`;
